hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/CapableOzoneFSOutputStream.java

```diff
@@ -38,12 +38,18 @@
  */
 public class CapableOzoneFSOutputStream  extends OzoneFSOutputStream
     implements StreamCapabilities {
-  public CapableOzoneFSOutputStream(OzoneFSOutputStream outputStream) {
+  private final boolean isEnabled;
```

Comment:
How about calling it `isHsyncEnabled`? `isEnabled` sounds like enabling/disabling everything.